Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
```

Comment:
### [dalli](https://github.com/petergoldstein/dalli)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.2.3 → 3.2.7

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/dalli/3.2.3/3.2.7)_]





<details>
<summary>Commits</summary>

A change of **33** commits. See the full changes on [the compare page](https://github.com/petergoldstein/dalli/compare/v3.2.3...v3.2.7).

These are the first 10 commits:
- (d81fac3) [Changes for new empty line before assert cop. (#935)](https://github.com/petergoldstein/dalli/commit/d81fac3d7cbe899a69cce8bcb97e27ed7563797a)
- (4fffe20) [Test with Ubuntu 20.04 to support build of memcached 15.x (#…](https://github.com/petergoldstein/dalli/commit/4fffe20a49d10903e31d177602f86c8e4039856a)
- (7380f63) [Preallocate the read buffer in Socket#readfull (#936)](https://github.com/petergoldstein/dalli/commit/7380f634b254c2fce4c9fc3ef170f72e5aa7b03f)
- (ec4fefe) [Update the CHANGELOG with preallocation entry](https://github.com/petergoldstein/dalli/commit/ec4fefede07786d01f722f9ef1b95522a8447bfc)
- (bcfe112) [Pass Encoding instances rather than encoding names as String…](https://github.com/petergoldstein/dalli/commit/bcfe112e47a2769f63b068b26615ff6dbe800754)
- (d7f0420) [Explicitly add 3.2 to the test matrix in preparation for its…](https://github.com/petergoldstein/dalli/commit/d7f04204ab54c6efc91a1906dbc621c07ec36821)
- (1a56832) [Fix Rubocop lints from Rubocop 1.42 (#944)](https://github.com/petergoldstein/dalli/commit/1a56832069c3d785f0c217b42bb71c7eb2e83a5c)
- (7dd0804) [Remove unnecessary rescue. (#945)](https://github.com/petergoldstein/dalli/commit/7dd0804e3dc14982464cbd81ed26c69d24f44571)
- (f63d299) [Add bin scripts and a description in the README (#946)](https://github.com/petergoldstein/dalli/commit/f63d2990956b2db23b308696f4e61e5d6e70cd8a)
- (636753b) [Add JRuby 9.4 to the CI matrix (#947)](https://github.com/petergoldstein/dalli/commit/636753b76cbe0a9790a1eff39074195eae37412a)


</details>